runtime/astar/src/precompiles.rs

```diff
@@ -121,16 +120,11 @@ pub type AstarPrecompiles<R, C> = PrecompileSetBuilder<
     (
         // Skip precompiles if out of range.
         PrecompilesInRangeInclusive<
-            // TODO: what is the range for precompiles sets 1 - ?
-            (AddressU64<1>, AddressU64<40951>),
+            // We take range as last precompile index, UPDATE this once new prcompile is added
```

Comment:
Actually, I thought to be more precise here - we have 3 distinct ranges, and IMO we should cover each one precisely.